connectors/src/connectors/intercom/lib/intercom_api.ts

```diff
@@ -0,0 +1,13 @@
+import { Client } from "intercom-client";
+
+export async function validateAccessToken(notionAccessToken: string) {
```

Comment:
`notionAccessToken` -> to replace. 

Also, you might not need this function at all (to be confirmed)

Reply:
🤦🏻‍♀️